fvm/src/kernel/default.rs

```diff
@@ -807,15 +845,23 @@ where
 
     #[cfg(feature = "m2-native")]
     fn install_actor(&mut self, code_id: Cid) -> Result<()> {
-        // TODO figure out gas
-        self.call_manager
+        let size = self
```

Comment:
We'll likely need something significantly more complicated, but this is a good start I guess.